saleor/graphql/shop/mutations/close_circuit_breaker.py

```diff
@@ -0,0 +1,56 @@
+import graphene
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.utils.module_loading import import_string
+
+from ....core.error_codes import ShopErrorCode
+from ....graphql.app.types import App
+from ....permission.enums import SitePermissions
+from ...core import ResolveInfo
+from ...core.doc_category import DOC_CATEGORY_SHOP
+from ...core.mutations import BaseMutation
+from ...core.types import ShopError
+
+# TODO: make breaker_board instance a singleton
+breaker_board = None
+if settings.ENABLE_BREAKER_BOARD:
+    from ....webhook.transport.synchronous.circuit_breaker.breaker_board import (
+        BreakerBoard,
+    )
+
+    breaker_board = BreakerBoard(
```

Comment:
We instantiate `BreakerBoard` in the same way in more than one place. Maybe we can extract this code so it's not repeated.